components/Cards/index.js

```diff
@@ -17,3 +17,71 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+// select the main dom node to attach our dynamic content
+
+const cards = document.querySelector(".cards-container");
+
+// send GET request using axios
+
+axios
+  .get(`https://lambda-times-backend.herokuapp.com/articles`)
+
+  .then(resolved => {
+    const articles = resolved.data.articles;
+    // Handles Success: here's where we get the results from server
+
+    //console.log("Handles success", articles);
+
+    // Use Object.keys to create array
+    const articleTopics = Object.keys(articles);
+    console.log("topics", articleTopics);
+
+    //You will be creating a component for each 'article' in the list.
+
+    // Use forEach to iterate through array created through Object.keys above
+    articleTopics.forEach(topic => {
+      // Use another forEach to add component for each 'article' in the list.
+      articles[`${topic}`].forEach(article => {
+        // Add to DOM
+        cards.append(createCards(article));
```

Comment:
Reese nice approach to get each of the object keys and iterate through them to get the articles object for that category... good problem solving. Another way to get the same outcome is to use a `for...in` loop which is checking for key values from your response like so... 

```
axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(response => {
    // articles variable to host the articles from response
    let articles = response.data.articles;
    // going through the keys of the articles object 
    for (topic in articles) {
       //  console.log(topic) you should get the articles "categories" === keys
      // . console.log(articles[topics]) you should get the array that that key(topic) is equal too and using that array we can iterate through it using a for each and appending our component from there
      articles[topic].forEach(article => {
        entry.appendChild(createCards(article));
      });
    }
  });
```